src/actions/pause.js

```diff
@@ -60,7 +77,14 @@ export function paused(pauseInfo: Pause) {
     frames = await updateFrameLocations(frames, sourceMaps);
     const frame = frames[0];
 
-    const scopes = await client.getFrameScopes(frame);
+    const frameScopes = await client.getFrameScopes(frame);
+    const scopes = !isGeneratedId(frame.location.sourceId)
```

Comment:
this can be its own function, as we do the same thing on lines 230-237

Reply:
this might avoid extra async/await cycle for generated sources -- notice 'await' in the middle of the conditional operator

Reply:
:+1: